examples/commands/stores.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+require_once dirname(__DIR__).'/bootstrap.php';
+
+use Doctrine\DBAL\DriverManager;
+use Doctrine\DBAL\Tools\DsnParser;
+use MongoDB\Client as MongoDbClient;
+use Symfony\AI\Store\Bridge\Clickhouse\Store as ClickhouseStore;
+use Symfony\AI\Store\Bridge\Local\CacheStore;
+use Symfony\AI\Store\Bridge\Local\InMemoryStore;
+use Symfony\AI\Store\Bridge\MariaDb\Store as MariaDbStore;
+use Symfony\AI\Store\Bridge\Meilisearch\Store as MeilisearchStore;
+use Symfony\AI\Store\Bridge\Milvus\Store as MilvusStore;
+use Symfony\AI\Store\Bridge\MongoDb\Store as MongoDbStore;
+use Symfony\AI\Store\Bridge\Neo4j\Store as Neo4jStore;
+use Symfony\AI\Store\Bridge\Postgres\Store as PostgresStore;
+use Symfony\AI\Store\Bridge\Qdrant\Store as QdrantStore;
+use Symfony\AI\Store\Bridge\SurrealDb\Store as SurrealDbStore;
+use Symfony\AI\Store\Bridge\Typesense\Store as TypesenseStore;
+use Symfony\AI\Store\Command\DropStoreCommand;
+use Symfony\AI\Store\Command\SetupStoreCommand;
+use Symfony\Component\Cache\Adapter\ArrayAdapter;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\ConsoleOutput;
+use Symfony\Component\DependencyInjection\ServiceLocator;
+use Symfony\Component\HttpClient\HttpClient;
+
+$factories = [
+    'cache' => static fn (): CacheStore => new CacheStore(new ArrayAdapter(), cacheKey: '_commands'),
+    'clickhouse' => static fn (): ClickhouseStore => new ClickhouseStore(
+        HttpClient::createForBaseUri(env('CLICKHOUSE_HOST')),
+        env('CLICKHOUSE_DATABASE'),
+        env('CLICKHOUSE_TABLE'),
+    ),
+    'mariadb' => static fn (): MariaDbStore => MariaDbStore::fromDbal(
+        DriverManager::getConnection((new DsnParser())->parse(env('MARIADB_URI'))),
+        'my_table_openai',
+        'my_index',
+    ),
+    'memory' => static fn (): InMemoryStore => new InMemoryStore(),
+    'meilisearch' => static fn (): MeilisearchStore => new MeilisearchStore(
+        http_client(),
+        env('MEILISEARCH_HOST'),
+        env('MEILISEARCH_API_KEY'),
+        '_commands',
+    ),
+    'milvus' => static fn (): MilvusStore => new MilvusStore(
+        http_client(),
+        env('MILVUS_HOST'),
+        env('MILVUS_API_KEY'),
+        env('MILVUS_DATABASE'),
+        '_commands',
+    ),
+    'mongodb' => static fn (): MongoDbStore => new MongoDbStore(
```

Comment:
Not sure this is working with the local version. I think it's an atlas only feature.

cc @GromNaN

Reply:
Nice catch, I'm removing it.

Reply:
So in theory, we could test vectors locally? 🤔

Reply:
Many thanks 🙏